test/integration/sanSocialDominance.js

```diff
@@ -0,0 +1,59 @@
+const { testFieldTypes } = require('../helper.js')
```

Comment:
The file should be named: `sanSocialDominanceTest.js`

Reply:
Yes, you are right, just renamed it